app/client/ui/WebhookPage.ts

```diff
@@ -104,17 +104,24 @@ const WEBHOOK_COLUMNS = [
     type: 'Text',
     label: t('Status'),
   },
+  {
+    id: 'vt_webhook_fc11',
+    colId: 'authorization',
+    type: 'Text',
+    label: t('Header Authorization'),
+  },
 ] as const;
 
 /**
  * Layout of fields in a view, with a specific ordering.
  */
 const WEBHOOK_VIEW_FIELDS: Array<(typeof WEBHOOK_COLUMNS)[number]['colId']> = [
   'name', 'memo',
-  'eventTypes', 'url',
-  'tableId', 'isReadyColumn',
-  'watchedColIdsText', 'webhookId',
-  'enabled', 'status'
+  'eventTypes', 'tableId',
```

Comment:
Can you explain why the order has changed? The comment mentions that the order matters here. Maybe it is outdated.

Reply:
Because it's the order on the frontend too and I find this order more logical. 
- Having the authorization parameter next to the url 
- Having the table id next to the eventTypes 
- etc ..  
But if you thinks differently we can change it

Reply:
No it looks fine. I'm just asking as this change wasn't mentioned anywhere (PR description or issue).